certbot/main.py

```diff
@@ -405,6 +406,65 @@ def _init_le_client(config, authenticator, installer):
 
     return client.Client(config, acc, authenticator, installer, acme=acme)
 
+def _delete_if_appropriate(config):
+    """Does the user want to delete their now-revoked certs?"""
+
+    display = zope.component.getUtility(interfaces.IDisplay)
+
+    if not (config.certname or config.cert_path):
+        raise errors.Error('At least one of --cert-path or --cert-name must be specified.')
+
+    if config.certname and config.cert_path:
+        # first, check if certname and cert_path imply the same certs
+        cert_path_implied_cert_name = cert_manager.cert_path_to_lineage(config)
```

Comment:
@ohemorange Do you mean that instead of `cert_path_implied_cert_name`, it should be the other way around aka `cert_name_implied_cert_path` (and computed via something like `cert_manager.lineage_for_certname(config, config.cert_name)`?